Language-Version-History.md

```diff
@@ -1,6 +1,6 @@
 # Features Added in VB Language Versions
 
-*NOTE:* See how to [specify the language version for your project here](https://docs.microsoft.com/en-us/dotnet/visual-basic/language-reference/configure-language-version)
+*NOTE:* See how to [specify the language version for your project here](/dotnet/visual-basic/language-reference/configure-language-version)
```

Comment:
We should keep the absolute link, so developers can navigate from the GitHub source easily. However, we should remove the locale:

```suggestion
*NOTE:* See how to [specify the language version for your project here](https://docs.microsoft.com/dotnet/visual-basic/language-reference/configure-language-version)
```